Projects/Embark/Sources/Viewables/Actions/AddressAutocompleteAction/AddressInput.swift

```diff
@@ -0,0 +1,66 @@
+import Flow
+import Foundation
+import Hero
+import Presentation
+import UIKit
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct AddressInput {
+	let placeholder: String
+	let textSignal = ReadWriteSignal<String>("")
+	private let setTextSignal = ReadWriteSignal<String>("")
+	let setIsFirstResponderSignal = ReadWriteSignal<Bool>(true)
+	let shouldReturn = Delegate<String, Bool>()
+	let didEditSignal = ReadWriteSignal<Void>(())
+
+	var text: String {
+		get {
+			return textSignal.value
+		}
+		set(newText) {
```

Comment:
do you only set with a `prefilled` value? In that case shouldn't `prefill` be optional. Then you don't really need this setter / getter with two different signals?